README.md

```diff
@@ -1 +1,168 @@
-# Hack-Summit
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [🛠 Walkthough](#walkthough)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 HakcSummit 2023 <a name="about-project"></a>
+
+**HackSummit 2023** is a capstone project for HTML CSS and JavaScript module at microverse
+
+## 🛠 Built With <a name="built-with"> HTML CSS & Java Script</a>
+
+- Video to walkthrough of this project can be found [here]("https://www.loom.com/share/fadb70b392754ee8ab23da8b3fff2f26")
+
+## 👁  Walkthough <a name="walkthough"> </a>
+
+Checkout the [video](https://www.loom.com/share/354d06ce5b2d4e83a8d2eef21ad1cd48) for the walkthough of this project.
+
+### Tech Stack <a name="tech-stack"></a>
+
+> HTML CSS & Java Script
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **Presents the Home page for HackSummit 2023**
+- **Presents the About page for HackSummit 2023**
+- **Displays Speakers List with more and less button functionality**
+- **Displays Mobile Menu**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> The live demo link to this project can be found at: 
+
+- [Live Demo Link](https://kalbek.github.io/Capstone-1-Hack-Summit/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Try to have these in your local machine
+
+- Git and GitHub
+- VS Code or any code editor
+
+### Prerequisites
+
+In order to run this project you need 
+- A web browser
+- Code editor
+
+### Setup
+
+Clone this repository to your desired folder:
+
+### Install
```

Comment:
- [ ] As the previous reviewer has requested that you add the required documentation and steps to clone the project into the desired folder in the "Setup" section. This will help users quickly and easily get started with the project.
To do this, you can provide instructions on how to clone the project from the repository into the desired folder, including any necessary dependencies or packages that need to be installed. Additionally, you can include information on how to run the project and any relevant configuration settings.
By providing this documentation, you can ensure that others can easily set up and run the project, making it more accessible and user-friendly.